src/analyze/analyze.h

```diff
@@ -18,7 +18,13 @@ typedef enum DotMode {
         DEP_REQUIRE,
 } DotMode;
 
+typedef enum CapabilityMode {
+        ARG_CAP_DEFAULT,
```

Comment:
we usually use a naming prefix for enum values that is common with the enum name (yes, this is not the case above with the DotMode stuff above, but that's rally old cold that noone so far cleaned up.)

I think "literal" would be better than "default" for this btw.

Hence maybe:

CAPABILITY_LITERAL and CAPABILITY_MASK?